app/views/returns/credits/ConvertedCreditsView.scala.html

```diff
@@ -55,20 +55,14 @@
 @layout(pageTitle = title(form, messages("converted.credits.title"), Some(messages("return.title.tail")))) {
 
     @formHelper(action = controllers.returns.credits.routes.ConvertedCreditsController.onSubmit(mode), 'autoComplete -> "off") {
-
-    @sectionHeader(messages("credits.caption"))
-    @heading("converted.credits.heading")
-    @para(paraContent)
-    @h2heading(messages("converted.credits.heading.2"), Some("converted-credits-h2"))
-
-       @if(form.errors.nonEmpty) {
+        @if(form.errors.nonEmpty) {
```

Comment:
There is also content missing for the error on this form on the screen shot, do we have that content yet?

Reply:
^both forms missing error content

Reply:
Yeah, I added those to the workflowy list